Formula/protozero.rb

```diff
@@ -0,0 +1,19 @@
+class Protozero < Formula
+  desc "Minimalistic protocol buffer decoder and encoder in C++"
+  homepage "https://github.com/mapbox/protozero"
+  url "https://github.com/mapbox/protozero/archive/v1.3.0.tar.gz"
+  sha256 "85f9238fa662ff06a1e364f1461846a9d377846274e7f98407307e31086cab2b"
+
+  def install
+    (include/"protozero").install Dir["include/protozero/*"]
```

Comment:
We've made something of a habit out of rejecting header-only formulae arguing that sort of situation often means people should be embedding in their projects rather than distributing globally. Not sure how other maintainers feel about this particular case.